bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1044,18 +1124,11 @@ protected void updateRunningAlarmProperties() {
         } else {
             newStringValue = "No running alarm";
         }
-        result.put("RunningAlarmProperties", newStringValue);
```

Comment:
Wasn't the intention of this to override this value if it was in the results? You now would call `onValueReceived` twice instead.

Reply:
Oh you maybe right. I will undo this one.

Reply:
I have undone this change.